packages/warehouses/package.json

```diff
@@ -12,14 +12,14 @@
         "@google-cloud/bigquery": "^7.2.0",
         "@lightdash/common": "^0.1425.3",
         "lodash": "^4.17.21",
-        "pg": "^8.11.3",
+        "pg": "^8.13.1",
         "pg-cursor": "^2.10.0",
         "snowflake-sdk": "~1.14.0",
         "ssh2": "^1.14.0",
         "trino-client": "0.2.6"
     },
     "devDependencies": {
-        "@types/pg": "^8.10.7",
+        "@types/pg": "^8.11.10",
```

Comment:
does this version need to match pg 8.13.1 ?

Reply:
its on latest version